### ISSUE TITLE
Add rada to the mapping of administrative levels

### ISSUE THREAD

**Issue by andkov** [kse-ua/ua-de-center#9]
Modify the script `./manipulation/ellis-ua-admin.R` so that its product `./data-private/derived/ua-admin-map.rds` includes `rada` as one of the levels of administrative units. 
Specifically, create columns `rada_code` and `rada_name` in the object `ds_admin`

![image](https://user-images.githubusercontent.com/2608073/184046155-92fd4ab7-853c-4b2a-a809-be1e5187825b.png)

@Tytser , you will need to procure the data table where settlements are mapped onto radas, but if I remember correctly from the last time we spoke you have it handy somewhere.

**Comment by Tytser** [comment]
@andkov Yes, I found an old classification with settlement mapped into radas and a comparison between old and new admin codes. So I will make `rada_code` and `rada_name` based on them

**Comment by Tytser** [comment]
додав код і назву ради, але через структуру вихідного файлу зі старою класифікацією не впевнений у правильності - треба ретельна перевірка

**Comment by andkov** [comment]
![image](https://user-images.githubusercontent.com/2608073/185616783-c3e2b953-0825-44d5-9b24-06a460076a03.png)

@Tytser, підкажіть будь ласка, да брати файл  зазначений на 35?

**Comment by Tytser** [comment]
@andkov перепрошую - забув додати цей файл на диск 
тепер в скрипті є лінк на файл з порівнянням
